spring-cloud-skipper-server-core/src/main/resources/application.yml

```diff
@@ -192,7 +192,7 @@ spring:
     baselineOnMigrate: true
     locations:
       - classpath:org/springframework/cloud/skipper/server/db/migration/{vendor}
-    check-location: false
```

Comment:
This is deprecated (named something else now) but the default is false here and on the new property - I yanked it.